main.rb

```diff
@@ -0,0 +1,19 @@
+require './person'
+require './student'
+require './teacher'
+
+def main
+  first = Person.new(14)
+  second = Student.new(18, 'Ruby')
+  teacher1 = Teacher.new('Jack', 45, 'Math')
```

Comment:
- [Optional] And here your data is displayed in disorder, and to fix that you just need to reorganize your data `teacher1 = Teacher. new( 45, 'Math', 'Jack')`. It depends on how you organize also your parameter in initialize, it can be `def initialize(age, specialization, name = 'Unknown', parent_permission = true)` 😁 

<img width="941" alt="Screenshot 2023-07-07 at 22 39 59" src="https://github.com/mohamedSabry0/school_library/assets/74501523/7436319d-bf26-4e3d-ac17-04d4028ceb87">